hahomematic/platforms/custom/cover.py

```diff
@@ -145,14 +152,20 @@ def is_closed(self) -> bool | None:
     def is_opening(self) -> bool | None:
         """Return if the cover is opening."""
         if self._e_direction.value is not None:
-            return str(self._e_direction.value) == CoverActivity.OPENING
+            return (
+                str(self._e_direction.value) == CoverActivity.OPENING
+                or self._e_direction.value == HdmCoverActivity.OPENING
```

Comment:
The first (original) evaluation does a cast to `str`. Not sure whether it is wise to cast the new evaluation to an `int` as that might raise an exception?

Reply:
mypy loves the cast to int